src/autoconf/IngressController.py

```
@@ -30,14 +31,38 @@ def _to_instances(self, controller_instance) -> List[dict]:
+                continue
+            for config_name, config_value in configmap.data.items():
+                global_config_data[config_name] = config_value
+        for secret in self.__corev1.list_namespaced_secret(watch=False, namespace=namespace).items:
+            if not secret.metadata.annotations or "bunkerweb.io/GLOBAL_CONFIG" not in secret.metadata.annotations:
+                continue
+            if not secret.data:
+                self._logger.warning(
+                    f"Ignoring blank secret {secret.metadata.name}",
+                )
+                continue
+            for key, value in secret.data.items():
```

Ruff (0.8.2)

Loop control variable `key` not used within loop body

(B007)